rex/resource.py

```diff
@@ -634,16 +634,35 @@ def coordinates(self):
 
         Returns
         -------
-        coords : ndarray
+        lat_lon : ndarray
             Array of (lat, lon) pairs for each site in meta
         """
-        if self._coords is None:
-            if 'coordinates' in self.h5:
-                self._coords = self._get_coords('coordinates', slice(None))
+        return self.lat_lon
+
+    @property
+    def lat_lon(self):
+        """
+        Extract (latitude, longitude) pairs
+
+        Returns
+        -------
+        lat_lon : ndarray
+        """
+        if self._lat_lon is None:
+            if 'coordinates' in self:
+                self._lat_lon = self._get_coords('coordinates', slice(None))
             else:
-                raise ResourceKeyError("'coordinates' is not a valid dataset!")
+                self._lat_lon = self.meta
+                lat_lon_cols = ['latitude', 'longitude']
+                for c in self.meta.columns:
```

Comment:
is the trickiness here just to check for columns that are "lat" instead of "latitude"? Might be more secure to do `if c.lower() in ('lat', 'latitude'):` but maybe our clients could think of something more creative like latit and long?

Reply:
I seriously can't remember, but I like that